test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -303,6 +305,85 @@ var _ = Describe("Mixedcpus", Ordered, func() {
 
 			})
 		})
+		When("Disabling mixedCpus in the performance profile", func() {
+			It("should causes all pods that uses shared CPUs to fail", func() {
+
+				By("Creating a deployment with one pod asking for a shared cpu")
+				rl := &corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("1"),
+					corev1.ResourceMemory: resource.MustParse("100Mi"),
+					sharedCpusResource:    resource.MustParse("1"),
+				}
+				p := makePod(ctx, testclient.Client, testutils.NamespaceTesting,
+					withRequests(rl),
+					withLimits(rl),
+					withRuntime(components.GetComponentName(profile.Name, components.ComponentNamePrefix)))
+				dp := deployments.Make(testutils.DeploymentTesting, testutils.NamespaceTesting,
```

Comment:
Why 2 namespaces? the namespace of the deployment should be the namespace of the pod

Reply:
These are not 2 namespaces. The first is the deployment name, second is the namespace.

Reply:
Done.